packages/block-library/src/gallery/edit.js

```diff
@@ -302,8 +302,10 @@ class GalleryEdit extends Component {
 					) }
 				>
 					{ images.map( ( img, index ) => {
+						const imgAlt = ( img.alt.length > 0 ) ? `${ img.alt }. ` : '';
```

Comment:
We need to make img.alt is defined e.g: `img.alt && img.alt.length > 0`. Currently, if I upload an image to a gallery using drag & drop the editor crashes because the image does not contain an alt text.

Reply:
Thanks @jorgefilipecosta. I've amended the commit to include this change. 

https://github.com/WordPress/gutenberg/blob/14b57c7ccdeee88cd04a68d3f7a2f3fa6c9e6a33/packages/block-library/src/gallery/edit.js#L304-L306

Sorry if I wasn't able to test the drag and drop. I'll be more careful the next time.